Collections/script.js

```diff
@@ -256,14 +256,137 @@ console.log(filterByAge(people, 30));
 
 // Zadanie 11.
 
+people.forEach(function(user) {
+    let userDescription = `The user's name is ${user.name} and the user is ${user.age} years old. The user works as ${user.job} and drives ${user.car}`;
+
+//uzywamy funkcji forEach, zeby przejść przez kazdy element tabeli people i nazywamy sobie kazdy element tabeli jako 'user' i informujemy funkcje, ze ma przejsc przez kazdy taki element 'user' i wyswietlic jego dane zgodnie z opisem
+
+if(people.car === "Ford Mustang" || people.car === "BMW 3 Series") {
+    userDescription += " which is fun car.";
+}
+else {
+    userDescription += " which is boring car.";
+}
+//ustalamy sobie warunki, dla których ma się dodawać kwestia tego czy to fajny, czy nudny samochód 
+
+
+console.log(userDescription);
+//dajemy console loga wewnątrz funkcji, zeby miec dostep do elementow
+
+});
+
+
 // Zadanie 12.
 
+people.forEach(function(user) {
+//znowu uzywamy sobie funkcji forEach, aby przejsc przez wszystkie elementy tablicy 'people'
+if(user.age >= 40 && user.gender === "man"){
+    let olderMen = `The user is a ${user.gender} and he is ${user.age} years old.`;
+//stawiamy warunek- JEŚLI age i gender spełniają warunkui, to dopiero wtedy wyświetl zmienną olderMen
+    console.log(olderMen);
+}
+});
+
+function menAgeSum(people){ //nawiązujemy sobie do tabeli people
+    let sumAge = 0; //tworzymy zmienną, do której przypiszemy zsumowany wiek
+    people.filter(function(user){
+        // if(user.age >= 40 && user.gender === "man"){
+        //     return true;
+        // }
+        //     return false; LUB
+
+    return user.age >= 40 && user.gender === "man"
+    }).forEach(function(user){ //tutaj miałam duuzy problem; po odfiltrowaniu userow zgodnie z wiekiem i płcią bierzemy kazdego i dodajemy jego wiek do zmiennej, ktora poki co wynosi 0, na początku zrobiłam if(warunki), ale chat podpowiedział mi, ze wtedy musialabym zwrócić true/ false i dopiero wtedy jeśli true to sumAge += user.age, sama bym pewnie na to nie wpadła...
+        sumAge += user.age;
```

Comment:
Nie jestem pewien, co to były za warunki 🤔

`.forEach` po prostu wykonuje określone instrukcje dla każdego argumentu z tablicy.